### ISSUE TITLE
Compiling DSDT fails

### ISSUE THREAD

**Issue by ronangaillard** [syscl/XPS9350-macOS#10]
Compiling DSDT with `Deploy.sh` fails : 

```[  OK  ]  Create ./DSDT.
[  OK  ]  Create ./DSDT/prepare.
[  OK  ]  Create ./DSDT/precompile.
[  OK  ]  Create ./DSDT/compile.
/dev/disk0 (internal, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:      GUID_partition_scheme                        *750.2 GB   disk0
   1:                        EFI EFI                     209.7 MB   disk0s1
   2:                  Apple_HFS Mac OS DD               749.3 GB   disk0s2
   3:                 Apple_Boot Recovery HD             650.0 MB   disk0s3

Enter EFI's IDENTIFIER, e.g. disk0s1: disk0s1
[  OK  ]  Mount disk0s1.
[  OK  ]  Copy untouch ACPI tables.
[ ---> ]  Disassembling tables...
[  OK  ]  Disassemble tables.
[  OK  ]  Search specification tables by syscl/Yating Zhou.
[ ---> ]  Patching DSDT.dsl
[  OK  ]  Fix ADBG Error.
[  OK  ]  Rename GFX0 to IGPU.
[  OK  ]  Fix USB _PRW.
[  OK  ]  IRQ Fix.
[  OK  ]  SMBus Fix.
[  OK  ]  AC Adapter Fix.
[  OK  ]  Add MCHC.
[  OK  ]  Fix _WAK Arg0 v2.
[  OK  ]  Add IMEI.
[  OK  ]  Fix Non-zero Mutex.
[  OK  ]  OS Check Fix.
[  OK  ]  Add audio Layout 1.
[  OK  ]  Rename B0D3 to HDAU.
[  OK  ]  Remove GLAN device.
[  OK  ]  iGPU TPMX to MEM2.
[  OK  ]  IMTR->TIMR, _T_x->T_x.
[  OK  ]  PPMC and PMCR combine together credit syscl.
[  OK  ]  Insert DMAC(PNP0200).
[  OK  ]  Make Device(MATH) load correctly in macOS.
[  OK  ]  SBTN->SLPB with correct _STA 0x0B.
[ ---> ]  Patching SSDT-14.dsl
[  OK  ]  Rename GFX0 to IGPU.
[ ---> ]  Patching SSDT-5.dsl
[  OK  ]  Rename GFX0 to IGPU.
[  OK  ]  Brightness fix (Haswell).
[  OK  ]  Rename B0D3 to HDAU.
[ ---> ]  Copying tables to precompile...
[  OK  ]  Copy tables to precompile.
[ ---> ]  Copying untouched tables to ./DSDT/compile...
[  OK  ]  Copy untouched tables to ./DSDT/compile.
[ ---> ]  Compiling tables...
[FAILED]  Compiling DSDT.
Compiler aborting due to parser-detected syntax error(s)
/Users/ronan/Downloads/syscl/XPS9350-macOS/DSDT/precompile/DSDT.dsl  13793:                     Arg1
Error    6126 -                                             syntax error, unexpected PARSEOP_ARG1 ^


Intel ACPI Component Architecture
ASL+ Optimizing Compiler version 20141107-64 [Jan  2 2015]
Copyright (c) 2000 - 2014 Intel Corporation

ASL Input:     /Users/ronan/Downloads/syscl/XPS9350-macOS/DSDT/precompile/DSDT.dsl - 36049 lines, 1151244 bytes, 17132 keywords

Compilation complete. 1 Errors, 0 Warnings, 0 Remarks, 0 Optimizations
```

**Comment by syscl** [comment]
@ronangaillard Thanks for the tip, will refine now...

syscl

**Comment by zeusk** [comment]
+1

I'm facing this too; is there a fix planned?

**Comment by syscl** [comment]
I know how to fix it, but since it's somewhat trivial, thus I have to fix other issues first.

syscl

**Comment by zeusk** [comment]
can you share the trivial fix? I'm stuck on a machine without audio or proper QE because DSDT patching fails :/

**Comment by syscl** [comment]
@zeusk @ronangaillard try the latest deploy.sh, I added the MDBG patch already.

syscl

**Comment by ronangaillard** [comment]
Hi syscl,

Thanks for the update !

Unfortunately it still fails : 

```  OK  ]  Mount disk0s1.
[  OK  ]  Copy untouch ACPI tables.
[ ---> ]  Disassembling tables...
[  OK  ]  Disassemble tables.
[  OK  ]  Search specification tables by syscl/Yating Zhou.
[ ---> ]  Patching DSDT.dsl
[  OK  ]  Fix ADBG Error.
[  OK  ]  Fix MDBG Error credit syscl.
[  OK  ]  Fix word field length Dword->Qword credit syscl.
[  OK  ]  Rename GFX0 to IGPU.
[  OK  ]  Fix USB _PRW.
[  OK  ]  IRQ Fix.
[  OK  ]  SMBus Fix.
[  OK  ]  AC Adapter Fix.
[  OK  ]  Add MCHC.
[  OK  ]  Fix _WAK Arg0 v2.
[  OK  ]  Add IMEI.
[  OK  ]  Fix Non-zero Mutex.
[  OK  ]  OS Check Fix.
[  OK  ]  Fix brightness keys F11, F12 function.
[  OK  ]  Add audio Layout 1.
[  OK  ]  Rename B0D3 to HDAU.
[  OK  ]  Remove GLAN device.
[  OK  ]  iGPU TPMX to MEM2.
[  OK  ]  IMTR->TIMR, _T_x->T_x.
[  OK  ]  PPMC and PMCR combine together credit syscl.
[  OK  ]  Insert DMAC(PNP0200).
[  OK  ]  Make Device(MATH) load correctly in macOS.
[  OK  ]  SBTN->SLPB with correct _STA 0x0B.
[ ---> ]  Patching SSDT-14.dsl
[  OK  ]  Rename GFX0 to IGPU.
[ ---> ]  Patching SSDT-5.dsl
[  OK  ]  Rename GFX0 to IGPU.
[  OK  ]  Brightness fix (Haswell).
[  OK  ]  Rename B0D3 to HDAU.
[ ---> ]  Copying tables to precompile...
[  OK  ]  Copy tables to precompile.
[ ---> ]  Copying untouched tables to ./DSDT/compile...
[  OK  ]  Copy untouched tables to ./DSDT/compile.
[ ---> ]  Compiling tables...
[FAILED]  Compiling DSDT.
/Users/ronan/dev/XPS9350-macOS/DSDT/precompile/DSDT.dsl  11734:             \_SB.PCI0.GLAN.GPEH ()
Error    6084 -                                                        Object does not exist ^  (\_SB.PCI0.GLAN.GPEH)

/Users/ronan/dev/XPS9350-macOS/DSDT/precompile/DSDT.dsl  35754:     Scope (_SB.PCI0.GLAN)
Error    6085 -                         Object not found or not accessible from scope ^  (_SB.PCI0.GLAN)

/Users/ronan/dev/XPS9350-macOS/DSDT/precompile/DSDT.dsl  35754:     Scope (_SB.PCI0.GLAN)
Error    6116 -                    Forward references from Scope operator not allowed ^  (_SB.PCI0.GLAN)

/Users/ronan/dev/XPS9350-macOS/DSDT/precompile/DSDT.dsl  35770:             Notify (GLAN, 0x02)
Error    6010 -                                               Internal compiler error ^  (could not map type)

/Users/ronan/dev/XPS9350-macOS/DSDT/precompile/DSDT.dsl  35770:             Notify (GLAN, 0x02)
Error    6058 -                                                          Invalid type ^  ([Reference] found, Notify operator requires [Device|Processor|Thermal])


Intel ACPI Component Architecture
ASL+ Optimizing Compiler version 20141107-64 [Jan  2 2015]
Copyright (c) 2000 - 2014 Intel Corporation

Unhandled encoded type: 1B
ASL Input:     /Users/ronan/dev/XPS9350-macOS/DSDT/precompile/DSDT.dsl - 36048 lines, 1151120 bytes, 17132 keywords

Compilation complete. 5 Errors, 126 Warnings, 143 Remarks, 235 Optimizations
[  OK  ]  Compile DptfTa.
[  OK  ]  Compile SaSsdt.
[ ---> ]  Copying SSDT-rmne.aml to ./DSDT/compile...
[  OK  ]  Copy SSDT-rmne.aml to ./DSDT/compile.
[ ---> ]  Installing SSDT-XPS13SKL.aml to ./DSDT/compile...```

**Comment by syscl** [comment]
@ronangaillard thks, will see what's wrong, since I'm still fixing other issues~

syscl

**Comment by syscl** [comment]
@ronangaillard try the latest update of deploy.sh

syscl

**Comment by syscl** [comment]
I tested the DSDT patch part after the fix:
```
[  OK  ]  Mount disk0s1.
[  OK  ]  Copy untouch ACPI tables.
[ ---> ]  Disassembling tables...
[  OK  ]  Disassemble tables.
[  OK  ]  Search specification tables by syscl/Yating Zhou.
[ ---> ]  Patching DSDT.dsl
[  OK  ]  Fix ADBG Error.
[  OK  ]  Fix MDBG Error credit syscl.
[  OK  ]  Fix word field length Dword->Qword credit syscl.
[  OK  ]  Rename GFX0 to IGPU.
[  OK  ]  Fix USB _PRW.
[  OK  ]  IRQ Fix.
[  OK  ]  SMBus Fix.
[  OK  ]  AC Adapter Fix.
[  OK  ]  Add MCHC.
[  OK  ]  Fix _WAK Arg0 v2.
[  OK  ]  Add IMEI.
[  OK  ]  Fix Non-zero Mutex.
[  OK  ]  OS Check Fix.
[  OK  ]  Fix brightness keys F11, F12 function.
[  OK  ]  Add audio Layout 1.
[  OK  ]  Rename B0D3 to HDAU.
[  OK  ]  iGPU TPMX to MEM2.
[  OK  ]  IMTR->TIMR, _T_x->T_x.
[  OK  ]  PPMC and PMCR combine together credit syscl.
[  OK  ]  Insert DMAC(PNP0200).
[  OK  ]  Make Device(MATH) load correctly in macOS.
[  OK  ]  SBTN->SLPB with correct _STA 0x0B.
[ ---> ]  Patching SSDT-14.dsl
[  OK  ]  Rename GFX0 to IGPU.
[ ---> ]  Patching SSDT-5.dsl
[  OK  ]  Rename GFX0 to IGPU.
[  OK  ]  Brightness fix (Haswell).
[  OK  ]  Rename B0D3 to HDAU.
[ ---> ]  Copying tables to precompile...
[  OK  ]  Copy tables to precompile.
[ ---> ]  Copying untouched tables to ./DSDT/compile...
[ ---> ]  Compiling tables...
[  OK  ]  Compiling DSDT.
```
Now, I have to refine some other place, good things will happen...soon :)

syscl

**Comment by ronangaillard** [comment]
With update I don't any more compilation errors but I still have this issue : 
```Enter EFI's IDENTIFIER, e.g. disk0s1: disk0s1
[  OK  ]  Mount disk0s1.
[  OK  ]  Copy untouch ACPI tables.
[ ---> ]  Disassembling tables...
[  OK  ]  Disassemble tables.
[  OK  ]  Search specification tables by syscl/Yating Zhou.
[ ---> ]  Patching DSDT.dsl
[  OK  ]  Fix ADBG Error.
[  OK  ]  Fix MDBG Error credit syscl.
[  OK  ]  Fix word field length Dword->Qword credit syscl.
[  OK  ]  Rename GFX0 to IGPU.
[  OK  ]  Fix USB _PRW.
[  OK  ]  IRQ Fix.
[  OK  ]  SMBus Fix.
[  OK  ]  AC Adapter Fix.
[  OK  ]  Add MCHC.
[  OK  ]  Fix _WAK Arg0 v2.
[  OK  ]  Add IMEI.
[  OK  ]  Fix Non-zero Mutex.
[  OK  ]  OS Check Fix.
[  OK  ]  Fix brightness keys F11, F12 function.
[  OK  ]  Add audio Layout 1.
[  OK  ]  Rename B0D3 to HDAU.
[  OK  ]  iGPU TPMX to MEM2.
[  OK  ]  IMTR->TIMR, _T_x->T_x.
[  OK  ]  PPMC and PMCR combine together credit syscl.
[  OK  ]  Insert DMAC(PNP0200).
[  OK  ]  Make Device(MATH) load correctly in macOS.
[  OK  ]  SBTN->SLPB with correct _STA 0x0B.
[ ---> ]  Patching SSDT-14.dsl
[  OK  ]  Rename GFX0 to IGPU.
[ ---> ]  Patching SSDT-5.dsl
[  OK  ]  Rename GFX0 to IGPU.
[  OK  ]  Brightness fix (Haswell).
[  OK  ]  Rename B0D3 to HDAU.
[ ---> ]  Copying tables to precompile...
[  OK  ]  Copy tables to precompile.
[ ---> ]  Copying untouched tables to ./DSDT/compile...
[  OK  ]  Copy untouched tables to ./DSDT/compile.
[ ---> ]  Compiling tables...
[  OK  ]  Compiling DSDT.
[  OK  ]  Compile DptfTa.
[  OK  ]  Compile SaSsdt.
[ ---> ]  Copying SSDT-rmne.aml to ./DSDT/compile...
[  OK  ]  Copy SSDT-rmne.aml to ./DSDT/compile.
[ ---> ]  Installing SSDT-XPS13SKL.aml to ./DSDT/compile...
[FAILED]  Copy SSDT-XPS13SKL.aml to ./DSDT/compile.```

**Comment by syscl** [comment]
@ronangaillard I see, because this SSDT hasn't implemented yet. Because I still tested this tiny SSDT(with some of my improvement, but I don't think it may be necessary).

syscl

**Comment by ronangaillard** [comment]
So if I go on I get this issue : 
```[ ---> ]  Installing audio...
Password:
[  OK  ]  Install audio.
[ Note ]  Set System Agent (SA) Configuration—>Graphics Configuration->DVMT Pre-Allocated->『160MB』.
[ ---> ]  Unlocking maximum pixel clock...
[FAILED]  Patch and sign framework.
/System/Library/Frameworks/CoreDisplay.framework/Versions/Current/CoreDisplay: replacing existing signature
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
/System/Library/Frameworks/CoreDisplay.framework/Versions/Current/CoreDisplay: the codesign_allocate helper tool cannot be found or used```

**Comment by ronangaillard** [comment]
And ```[ ---> ]  Leading to lid wake on 0x19260004 (c) syscl/lighting/Yating Zhou...
[  OK  ]  Lead to lid wake on 0x19260004.
[  OK  ]  Generate configuration file of sleepwatcher launch daemon.
[  OK  ]  Generating script to unmount external devices before sleep (c) syscl/lighting/Yating Zhou.
[  OK  ]  Generate script to load RTWlanUSB upon sleep.
[ ---> ]  Installing external devices sleep patch...
[  OK  ]  Install sleepwatcher daemon.
[  OK  ]  Install configuration of sleepwatcher daemon.
[  OK  ]  Install sleep script.
[  OK  ]  Install wake script.
[  OK  ]  Fix the permissions of /etc/sysclusbfix.sleep.
[  OK  ]  Fix the permissions of /etc/sysclusbfix.wake.
[  OK  ]  Trigger startup service of syscl.usb.fix.
[  OK  ]  Clean up.
[  OK  ]  Mount disk0s4.
[  OK  ]  Create /tmp/RecoveryHD.
[  OK  ]  Create /Users/ronan/dev/XPS9350-macOS/Backups/2016-12-12-h18_09_15.
[ ---> ]  Convert UDZO(r/o) to UDRW(r/w) ...
[  OK  ]  Convert UDZO(r/o) to UDRW(r/w).
[  OK  ]  Attach Recovery HD.
[FAILED]  Patch and sign framework for Recovery HD.```

**Comment by gctseng** [comment]
Main question.  Can I just run the Deploy.sh again after updating my local git repository?

Below is some information for you from running Deploy.sh before your fixes above (I will run the script again depending on your answer above).  A full copy of the log 
[2016-12-11_2332 Dell XPS 13 9350 Deploy_sh log.txt](https://github.com/syscl/XPS9350-macOS/files/646957/2016-12-11_2332.Dell.XPS.13.9350.Deploy_sh.log.txt)
[2016-12-11_2332 Dell XPS 13 9350 Deploy_sh log.txt](https://github.com/syscl/XPS9350-macOS/files/646961/2016-12-11_2332.Dell.XPS.13.9350.Deploy_sh.log.txt)

is attached.  I thought it was because I still have the DW1820 (I haven't, and won't bother making it work--I'm waiting for my DW1830).

First error, which you probably fixed based on comment above.

`[ ---> ]  Updating files from https://github.com/syscl/XPS9350-macOS...
[ Note ]  https://github.com/syscl/XPS9350-macOS is not available at this time, please link https://github.com/syscl/XPS9350-macOS again next time..
[  OK  ]  Create ./DSDT.
...
[  OK  ]  Copy untouched tables to ./DSDT/compile.
[ ---> ]  Compiling tables...
[FAILED]  Compiling DSDT.
/Users/gctseng/git/XPS9350-macOS/DSDT/precompile/DSDT.dsl  11734:             \_SB.PCI0.GLAN.GPEH ()
Error    6084 -                                                          Object does not exist ^  (\_SB.PCI0.GLAN.GPEH)

/Users/gctseng/git/XPS9350-macOS/DSDT/precompile/DSDT.dsl  35755:     Scope (_SB.PCI0.GLAN)
Error    6085 -                           Object not found or not accessible from scope ^  (_SB.PCI0.GLAN)

/Users/gctseng/git/XPS9350-macOS/DSDT/precompile/DSDT.dsl  35755:     Scope (_SB.PCI0.GLAN)
Error    6116 -                      Forward references from Scope operator not allowed ^  (_SB.PCI0.GLAN)

/Users/gctseng/git/XPS9350-macOS/DSDT/precompile/DSDT.dsl  35771:             Notify (GLAN, 0x02)
Error    6010 -                                                 Internal compiler error ^  (could not map type)

/Users/gctseng/git/XPS9350-macOS/DSDT/precompile/DSDT.dsl  35771:             Notify (GLAN, 0x02)
Error    6058 -                                                            Invalid type ^  ([Reference] found, Notify operator requires [Device|Processor|Thermal])

Intel ACPI Component Architecture
ASL+ Optimizing Compiler version 20141107-64 [Jan  2 2015]
Copyright (c) 2000 - 2014 Intel Corporation

Unhandled encoded type: 1B
ASL Input:     /Users/gctseng/git/XPS9350-macOS/DSDT/precompile/DSDT.dsl - 36049 lines, 1151161 bytes, 17133 keywords

Compilation complete. 5 Errors, 126 Warnings, 143 Remarks, 235 Optimizations`

Second error (noted your comment above):

`[ ---> ]  Installing SSDT-XPS13SKL.aml to ./DSDT/compile...
[FAILED]  Copy SSDT-XPS13SKL.aml to ./DSDT/compile.
cp: /Users/gctseng/git/XPS9350-macOS/DSDT/prepare/SSDT-XPS13SKL.aml: No such file or directory
[  OK  ]  Clean dynamic SSDTs.`

Lastly:

`[  OK  ]  Convert UDRW(r/w) to UDZO(r/o).
[ ---> ]  Updating Recovery HD for DELL M3800/XPS9530...
cp: /Volumes/Recovery HD/com.apple.recovery.boot/BaseSystem.dmg: Permission denied
[  OK  ]  Clean redundant dmg files.
[  OK  ]  Unmount disk0s3.
...
[ Note ]  Reboot now. Then enjoy your OS X! -syscl/lighting/Yating Zhou @PCBeta.`

**Comment by ronangaillard** [comment]
Running `./Deploy.sh` not looking at any issue results in an unbootable system.

![dsc_0155](https://cloud.githubusercontent.com/assets/5607736/21115487/bebeff28-c0b1-11e6-91cb-670c508c4208.jpg)

**Comment by syscl** [comment]
I'm working on figuring out this issue for a few days, that's the onbroad(hda-gfx) inject issue, but still I have no clues about it. Though it seems no effect, but I double this cause no hdmi/dp device in system report-> audio

I'm outside, will be back tonight,
syscl

**Comment by applesjgtl** [comment]
Still failing to compile. My fresh install won't boot at all from your CLOVER folder. I have a DW1550 which I've been unable to activate (ended up removing those drivers as the boot was hanging there) but now it hangs elsewhere.

**Comment by ronangaillard** [comment]
@applesjgtl this is an unrelated issue, maybe you should create a new issue.

**Comment by applesjgtl** [comment]
@ronangaillard I may not have been clear enough - DSDT is still failing to compile. For sure there are other issues though... May or may not be related.

**Comment by ronangaillard** [comment]
@applesjgtl do you manage to boot using the clover folder from this repo (before using `deploy.sh` or compiling any DSDT) ?

**Comment by zeusk** [comment]
I think we should keep the comments related to the issue.

Also, I was able to boot with the in-repo CLOVER last week (haven't tested latest but there was only one change since 37d5223). You could try reverting that and booting then but I suggest creating a new ticket for that.

**Comment by applesjgtl** [comment]
@ronangaillard Does not boot with clover folder from this repo, hangs around the same place as above photo

**Comment by ronangaillard** [comment]
@applesjgtl This is another issue then, you should create a new one :)

**Comment by gctseng** [comment]
@applesjgtl Start a new thread and we can discuss.

As a data point, I can confirm I was able to boot with the CLOVER folder from this repo to install Sierra 10.12.1 (made USB using 54c47c6 on 2016-12-09 and Clover_v2.3k_r3949). I noticed the onscreen log said you have a duplicate kext. I deleted the CLOVER folder that was installed on the USB by Clover installer before I copied the CLOVER folder over.

**Comment by gctseng** [comment]
Just ran Deploy.sh and no compile errors.  Actually no errors except for the SSDT-XPS13SKL.aml issue discussed above. Audio issue discussed in other thread, but just wanted to note it boots!

**Comment by ronangaillard** [comment]
@gctseng What specs do you have ?
Because I still have issues and 10.12.1 won't boot...

**Comment by gctseng** [comment]
OS X 10.12.1, QHD+ (touchscreen enabled), i7-6560U, 16GB, 512GB NVMe THNSN5512GPUK NV, BIOS 1.4.10

Can you boot using USB?

**Comment by ronangaillard** [comment]
Hummm same specs here... No I cannot boot neither by USB or using SSD...

Weird !

**Comment by gctseng** [comment]
Maybe it's a bios settings issue? Other than updating my bios to 1.4.10 before I installed OS X (using the CLOVER version on 12/9), I followed the bios settings mentioned here:
[https://www.tonymacx86.com/threads/guide-dell-xps-13-9350-macos-10-12-1.204730/](url)

I should mention that I still have the stock DW1820 (waiting on DW1830) but I have a USB/Ethernet/hub dongle on which I am using for the Internet.

I should also mention that my laptop can't see the USB key to boot from it anymore (happened right after the install, you'll see my detailed description on the tonymac thread). I may just recreate the USB key because if I somehow get stuck I need to have a reliable way to still boot.

**Comment by syscl** [comment]
@ronangaillard What the error? I double check this morning: I compile the ACPI tables by deploy.sh and use it for my laptop, it's bootable.

BTW, if you can try to make a clean build by the following(under XPS9350-macOS/)
```sh
tools/cleanup
```
Then use deploy.sh again.

syscl

**Comment by icedman** [comment]
suggestion:

1. update to latest iasl version
2. try disassembling with :

> iasl -da -dl -fe refs.txt *.aml

with ref.txt: 

External(MDBG, MethodObj, 1)
External(_GPE.MMTB, MethodObj, 0)
External(_SB_.PCI0.LPCB.H_EC.ECWT, MethodObj, 2)
External(_SB_.PCI0.LPCB.H_EC.ECRD, MethodObj, 1)
External(_SB_.PCI0.PEG0.PEGP.SGPO, MethodObj, 2)
External(_SB.PCI0.GFX0.DD02._BCM, MethodObj, 1)
External(_SB.PCI0.SAT0.SDSM, MethodObj, 4)
External(_SB.PCI0.SAT1.SDSM, MethodObj, 4)
External(_GPE.VHOV, MethodObj, 3)

or..
https://www.tonymacx86.com/threads/guide-patching-laptop-dsdt-ssdts.152573/

**Comment by syscl** [comment]
@ronangaillard @zeusk @gctseng @applesjgtl @icedman hi everybody, finally fix the HDMI/DP issue credit myself...

Here's the proof, thus you know what I am doing now;)
![snip20161213_1](https://cloud.githubusercontent.com/assets/11594858/21165069/89761eda-c162-11e6-8b4d-f3b86db03831.png)

Cheers,
syscl

**Comment by gctseng** [comment]
@syscl Is this fix on the repository now? Should I wait to rebuild and deploy until after you upload it?

**Comment by syscl** [comment]
@gctseng wait for my update.

syscl

**Comment by riptidewave93** [comment]
Just reporting that as of the current repo revision, the build issue is fixed for me.

**Comment by syscl** [comment]
@riptidewave93 right, it has been solved already.

syscl
